### DISCUSSION TITLE
Masked Background Blur Effect

### DISCUSSION THREAD

**Discussion by Adhil-Anzar** [AvaloniaUI/Avalonia#17200] [Q&A] [answered]
How can I implement something of this sort? 
![properties](https://github.com/user-attachments/assets/33442e3e-11c5-4980-9f15-d5ac28ad6ff5)
Where a visual exists and there exist another visual which is clipped ( yes ik you can duplicate clip and then add Effect="blur(10)" ) but is there a better way to do this?

**Comment by timunie** [comment]
Check out Aura.UI (find it on awsome-avalonia). They have something similar iirc. You may need to port it to 11.0

**Reply by Adhil-Anzar** [reply]
I tried it but idts it is possible...
It only works for Images
Thanks for the response

**Reply by StefanKoell** [reply, marked as answer]
@Adhil-Anzar you may checkout this one (quickly hacked together):
https://gist.github.com/StefanKoell/b7dd1c847984a9a9ae75ed4a96fbc4b5

Ensure you add the SimpleBlur.Styles.axaml and you should be good to go.

**Reply by Adhil-Anzar** [reply]
Thank you Sir, It worked Perfectly
